src/foam/core/Context.js

```diff
@@ -43,8 +43,22 @@
      * @param opt_suppress Suppress throwing an error.
      **/
     lookup: function(id, opt_suppress) {
-      var ret = typeof id === 'string' && this.__cache__[id];
-
+      // If it's an interface extending other interfaces, it won't
```

Comment:
I don't think this should be handled here. Lookup should only support a String id. Supporting arrays of extends values should be handled in the interface code, not here.